bin/reset-all

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+yarn docker:stop
+
+docker volume prune
```

Comment:
These two could be combined as `yarn docker:stop --volumes` which would also have the benefit on not removing unrelated volumes.